readyapi4j-core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java

```diff
@@ -85,6 +89,144 @@ public void buildsRecipeWithPropertyTransferTestStep() throws Exception {
     }
 
     @Test
+    public void buildsJsonPathTransferWithImplicitSourceStep() throws Exception {
+        final String jsonPath = "$.customer.address";
+        TestRecipe recipe = newTestRecipe()
+                .addStep(getRequest("/get/something").named("theGet"))
+                .addStep(propertyTransfer(fromPreviousResponse(jsonPath)
```

Comment:
Could we distill this to `newTestRecipe( TestStepBuilder ... testSteps )` - resulting in

```
newTestRecipe( 
   getRequest(...),
   propertyTransfer( ... ),
  ...
)
```

?